setup.py

```diff
@@ -26,4 +26,8 @@
 
 setuptools.setup(
     setup_requires=['pbr>=2.0.0'],
+    extras_require={
```

Comment:
I have no experience with pbr, but I think these need to be added to the `setup.cfg` https://docs.openstack.org/pbr/3.1.0/index.html#extra-requirements

Reply:
Wow, indeed, there is pbr. Fixed.